"""
exceptions.py

Содержит пользовательские исключения для парсера.
"""

class ParserError(Exception):
    """Исключение, возникающее при ошибках парсинга."""
    pass
